diff.py

```diff
@@ -1446,6 +1446,12 @@ def run_less(self) -> "Tuple[subprocess.Popen[bytes], subprocess.Popen[bytes]]":
             header_lines = [header] if header else []
             output = "\n".join(header_lines + diff_lines[args.skip_lines :])
 
+        # Windows Terminal does not handle buffers properly
+        # Janky hack to clear its scrollback buffer until it's fixed
+        clear_proc = subprocess.Popen(
+            ["echo", "-en", "\"\e[3J\""]
+        )
```

Comment:
Popen() starts a new process that runs asynchronously; we'd need to wait for it to finish if we do this (clear_proc.wait() or subprocess.run()/similar). Does it work if you instead do
```py
        sys.stdout.write("\x1b7[3J\"")
        sys.stdout.flush()
```
?

There is a `os.system("tput reset")` further down which is similar to this btw, but I guess it doesn't help for whatever reason (and it doesn't do anything for the very first less call).

Reply:
The stdout writes don't work unfortunately.